### ISSUE TITLE
Option to Lock Sizes within a Group

### ISSUE THREAD

**Issue by AdityaPGit** [excalidraw/excalidraw#8916]
Currently, when a group of items is scaled, all items inside the group are scaled proportionally. There is no feature to lock the dimensions of specific items within the group.

Feature Request:
Add an option to lock the dimensions of individual items in a group so they remain unchanged during group scaling. This would provide more flexibility when working with grouped elements, especially when certain items need to maintain their original size.

**Comment by Ceredril** [comment]
This seems to be the same as #2767. I am not sure what "out of scope" means in this case. I could really use this feature.

**Comment by AdityaPGit** [comment]
> This seems to be the same as #2767. I am not sure what "out of scope" means in this case. I could really use this feature.

Exactly. I happen to need this feature but discovered its not yet implemented. I'm suprised how it is marked as "out of the scope"
